src/main/java/com/onarandombox/MultiverseCore/utils/player/checkresult/BlacklistResult.java

```diff
@@ -0,0 +1,32 @@
+package com.onarandombox.MultiverseCore.utils.player.checkresult;
+
+import com.onarandombox.MultiverseCore.api.action.SimpleActionResult;
+
+public class BlacklistResult extends SimpleActionResult {
```

Comment:
Hmm... I'm getting kind of hung up on these class names. BlacklistResult makes me think there is some kind of blacklist action that we're getting the result for. But it's not a blacklist action, it's a permission check action or similar we're trying to get the result for.

Reply:
actually yea that is possible

Reply:
But thinking again, I think having a specific return type like `BlacklistResult` is better than `OperationResult`, so users know exactly only these possible results will be returned, not any other results.

Reply:
Hmm yeah, that's fair.